web/app/templates/base.html

```diff
@@ -1,8 +1,9 @@
 {% extends "bootstrap/base.html" %}
 {% block styles%}
 {{ super() }}
+<link href="https://cdnjs.cloudflare.com/ajax/libs/jqueryui/1.10.0/css/smoothness/jquery-ui-1.10.0.custom.min.css" rel="stylesheet" type="text/css">
 <link href="{{ url_for('static', filename='css/chronos-bootstrap.sass.css') }}" rel="stylesheet" type="text/css">
-<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.2/jquery.min.js"></script>
+<link href="{{ url_for('static', filename='css/slider-style.sass.css') }}" rel="stylesheet" type="text/css">
```

Comment:
Do we really need `slider-style` and `jQuery UI` for every page we load?

I think this only should be loaded in the templates that require the slider component